remoteworkitem/trackerquery_repository_test.go

```diff
@@ -73,6 +82,7 @@ func (test *TestTrackerQueryRepository) TestExistsTrackerQuery() {
 		req := &http.Request{Host: "localhost"}
 		params := url.Values{}
 		ctx := goa.NewContext(context.Background(), nil, req, params)
+		testFxt := tf.NewTestFixture(test.T(), test.DB, tf.Spaces(1))
```

Comment:
You should really use the shorthand `t` that is created in the beginning of this file. Or maybe you can follow the convention of calling the suite receiver `s` as done in most tests. Then you would write `s.T()`.

Oh, hold on, you MUST use `t` here but that will not evaluate to the shorthand from the beginning of the function but the `t` from the subtest that you're in.

Reply:
done here 
cb373d2

Reply:
There used to be a lot more places where the `t` was not used but maybe I've brought that up in a different comment. So I resolve this one.